utils/browser/Cargo.toml

```
@@ -0,0 +1,27 @@
+[package]
+name = "browser-utils"
+version = "2.0.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "Utilities for creating a browser light-client."
+edition = "2018"
+
+[dependencies]
+futures = "0.3"
+futures01 = { package = "futures", version = "0.1.29" }
+log = "0.4.8"
+libp2p = { version = "0.13.1", default-features = false }
+console_error_panic_hook = "0.1.1"
+console_log = "0.1.2"
+js-sys = "0.3.22"
+wasm-bindgen = "0.2.55"
+wasm-bindgen-futures = "0.4.5"
+kvdb-web = "0.1.1"
+service = { package = "sc-service", path = "../../client/service", default-features = false }
```

will need a version now. The following two lines, too.
```suggestion
service = { version = "2.0.0", package = "sc-service", path = "../../client/service", default-features = false }
```